src/server/types/newrelic_pino-enricher.d.ts

```diff
@@ -0,0 +1 @@
+declare module "@newrelic/pino-enricher";
```

Comment:
Do we need this?

Reply:
this is to suppress `Could not find a declaration file for module '@newrelic/pino-enricher'. ` error. there is no type declaration in @types